src/App.js

```diff
@@ -7,9 +7,25 @@ import Login from 'views/login';
 import 'bootstrap/dist/css/bootstrap.css';
 import 'react-select/dist/react-select.css';
 import './App.css';
+import { BitGo } from 'bitgo';
+
+function getBitgoInstanceFromEnv() {
+  const env = process.env['REACT_APP_BITGO_ENV'];
+  const token = process.env['REACT_APP_BITGO_TOKEN'];
+  if (!token || !env) {
+    return undefined;
+  }
+  return new BitGo({ env, accessToken: token });
+}
 
 class App extends Component {
-  state = { isLoggedIn: false, loginBypass: false, bitgo: null };
+  constructor() {
+    super();
+    const bitgo = getBitgoInstanceFromEnv();
+    this.state = bitgo
```

Comment:
What's this for? Doesn't seem relevant to this PR but I do see some calls around the repo that pull `Bitgo` from the state. How did it work before?

Reply:
Looks like `bitgo` instance was assigned during login, stored in the `this.props` object. For example, here is how `bitgo` is assigned for nonBitGo features https://github.com/BitGo/wallet-recovery-wizard/blob/1d29e4812adeff2119f14e7595f46266ccba1da5/src/views/login/index.js#L44  

Although I am not sure how `bypassLogin` does the actual assignment. I don't see any function definition besides 
```
      return <Login finishLogin={this.updateLoginState(true)} bypassLogin={this.updateLoginBypass} {...props} />;
``

Reply:
Actually, `state` is more appropriate here. We do mutate `bitgo` at some point in non-bitgo

Reply:
this change allows me to set the `REACT_APP_` envars in order to skip the login screen and have a valid bitgo instance

Reply:
since `bitgo` was a `state` property before I didn't change it (just changed the location where `state` is set)

Reply:
ah gotcha, cool 👍